tests/Fixtures/inc/Engine/Cache/WPCache/addWpCacheStatusTest.php

```diff
@@ -2,7 +2,24 @@
 
 return [
     'test_data' => [
+        'testShouldBailOutWhenSetCacheConstFilterFalse' => [
+            'config' => [
+				'set_filter_to_false'   => false,
+			],
+			'tests'   => [],
+			'expected' => [
+                'direct' => [
+                    'wp_cache_status' => [
+                        'label' => 'WP_CACHE value',
+                    ],
+                ],
```

Comment:
```suggestion
```

In this case we don't expect the original value to be modified, since we bail-out early